dnsutil.go

```diff
@@ -214,21 +214,11 @@ func checkDNSPropagation(fqdn, value string, resolvers []string) (bool, error) {
 		fqdn += "."
 	}
 
-	// Initial attempt to resolve at the recursive NS
-	r, err := dnsQuery(fqdn, dns.TypeTXT, resolvers, true)
-	if err != nil {
-		return false, err
-	}
-
 	// TODO: make this configurable, maybe
 	// if !p.requireCompletePropagation {
 	// 	return true, nil
 	// }
 
-	if r.Rcode == dns.RcodeSuccess {
-		fqdn = updateDomainWithCName(r, fqdn)
-	}
-
```

Comment:
Are these removals necessary for getting the OverrideDomain feature to work? And does it still make sense for the commented block to remain if the surrounding code is now gone?

If these lines aren't required in general, I guess it's OK to remove them, but ideally I'd rather make them opt-in I think (configurable).